fetch-todolist/src/services/api.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+const instance = axios.create({ baseURL: "http://localhost:8080" });
```

Comment:
can we configure server url http://localhost:8080 from .env

Reply:
yes we can do it by installing dotenv and then setting process.env.CI  to false